### ISSUE TITLE
@codemirror v0.18 in @codesandbox/sandpack-react breaks when targeted for es5

### ISSUE THREAD

**Issue by JayaKrishnaNamburu** [codesandbox/sandpack#22]
### Error

With @codemirror v0.18.0 builds break, which is being used from `react-smooshpack-1.0.0-beta-0`. Since, the `@codemirror/autocomplete` uses [Nullish Coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator). And it is not transpiled when published to npm.

I am not sure, if `codemirror` should handle this or the end user. But most of the build tools, don't transpile their dependencies. And so, builds break when we transplie projects and not target **es6** at the end.

```js
new RegExp(`${addStart ? "^" : ""}(?:${source})${addEnd ? "$" : ""}`, expr.flags ?? (expr.ignoreCase ? "i" : ""));
```

**Comment by JayaKrishnaNamburu** [comment]
Not sure, where this can be handled. But just opened it here. So, you can decide if `sandack-react` highlights in the docs or something.

**Comment by alexnm** [comment]
I can try fixing the version to 0.18.0 for all codemirror packages. The null coalescing operator should be transpiled for the es5 build. I also noticed this break in next.js for the server render

**Comment by JayaKrishnaNamburu** [comment]
Yes, even fix to `v0.17` since with `^0.18.0` in dependencies. It still pulls down `v0.18.1` during npm install. And in `0.18.1` it is having this issue.

**Comment by alexnm** [comment]
I mean fixing to `0.18.0`, not `^0.18.0`. v0.17 has its issues as well :P it's not the first time the library broke something in sandpack

**Comment by alexnm** [comment]
appears to be fixed in 0.18.2. I will probably block the dependencies for codemirror in the future, but for now, it should work with removing yarn.lock and reinstalling our package to v0.0.1
